docs/src/components/App/Header/Header.less

```diff
@@ -129,8 +129,10 @@
   color: white;
   text-decoration: none;
   display: inline-block;
-  margin-bottom: @grid-row-height;
   &:hover {
     box-shadow: inset 0 -2px 0 @sk-white;
   }
+  @media @desktop {
+    margin-bottom: (@row-height * 2);
+  }
```

Comment:
This doesnt look right on the docs site, the links in the menu on desktop appear too loose. I'd just remove it.